language/move-prover/bytecode-to-boogie/src/spec_translator.rs

```diff
@@ -104,23 +104,6 @@ impl<'env> SpecTranslator<'env> {
             })
             .join(" || ");
 
-        // negation of abort_if condition is an implicit succeeds_if condition.
```

Comment:
This is what I wanted the code to do.  Is it causing problems with specifications?